front/src/Factur.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export default (facture : any ) => {
+  return (
+    <div className='facturs' style={{ display: 'flex', justifyContent: 'center'}}>
```

Comment:
inline style. Zrób sobie jakąś klasę css (unikaj stalowania in-line) albo zastosuj styled-components.

Reply:
dziwne bo nie widzę tego w kodzie